test/e2e/eventListener/eventListener_test.go

```diff
@@ -0,0 +1,50 @@
+// +build e2e
+// Copyright © 2020 The Tekton Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package pipeline
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/tektoncd/cli/test/cli"
+	"github.com/tektoncd/cli/test/framework"
+	"github.com/tektoncd/cli/test/helper"
+	"gotest.tools/v3/assert"
+	knativetest "knative.dev/pkg/test"
+)
+
+const (
+	teEventListenerName = "listener"
+)
```

Comment:
The build test is failing due to having this const. It's not used so it should be removed:

```
Linting go files...
test/e2e/eventListener/eventListener_test.go:30:2: `teEventListenerName` is unused (deadcode)
	teEventListenerName = "listener"
```

Reply:
Oh, I forgot to remove it :grimacing:

Reply:
Done :+1: 
Thank you :pray: